### DISCUSSION TITLE
Error al correr los test con Maven

### DISCUSSION THREAD

**Discussion by yasminelias** [dds-jv/foro#210] [Consultas de herramientas] [answered]
Hola buen dia! Estaba tratando de correr los tests con Maven para hacer la entrega del TP1 y cuando quise hacerlo me da este error. Yo cuando lo probe el maven me andaba, no se que puede ser.
Gracias!
![Capture](https://user-images.githubusercontent.com/48914301/116405611-3953de80-a806-11eb-97c2-4eb3a01f5b35.JPG)

**Comment by gastonprieto** [comment, marked as answer]
Buenas @yasminelias !

Maven te anda, pero fijate lo que aparece en rojo en el error, dice que no encuentra el compilador, entonces no esta asociada la jdk correctamente. fijate que por acá hay un par de issues que comentan algunos problemas que fueron teniendo

**Reply by gastonprieto** [reply]
te sirvio @yasminelias ?

**Reply by yasminelias** [reply]
Hola, si perdón que no respondí, pensé que lo había hecho! Muchas gracias!!!!!!
